subsys/bluetooth/audio/ascs.c

```diff
@@ -27,6 +27,8 @@ LOG_MODULE_REGISTER(bt_ascs, CONFIG_BT_ASCS_LOG_LEVEL);
 #include "common/bt_str.h"
 #include "common/assert.h"
 
+#include "../host/att_internal.h"
+
```

Comment:
I agree, but this will require significant changes to ATT. I'll rather do that in a separate PR. If that PR merges before this, I'll update this PR; vice versa if this merges first I'll modify this in that PR.